news/2018-07-16-cwc-2018-recap-grand-finals.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "osu!catch World Cup 2018: Grand Finals Recap"
+date: 2018-07-16 09:00:00 +0000
+---
+
+Of our three fruit-obsessed finalists, only one team could be crowned this year's osu!catch World Cup Champions. Chile, China and the United States – through the fire and the falling, which would reign supreme? All the juicy details can be found within!
+
+![](/wiki/shared/news/banners/CWC_2018_banner.jpg)
+
+Greetings once again everyone, [JBHyperion](https://osu.ppy.sh/users/4879508) here for your final osu!catch World Cup 2018 update. The sun has set on the year's premiere fruit-catching tournament at last, and a team of worthy Champions have been crowned.
+
+It was truly a tournament for the ages, as old Gods fell, and new warriors arose to forever etch their names in history. There were ties and tiebreakers aplenty. Some crept by on the narrowest of margins, whilst others, sadly, fell agonizingly short. Needless to say, my expectations have been exceeded on almost every conceivable level!
+
+There's no need to simply take my word for it though – naturally you can find videos of both Finals Week 2 matches (and previous' weeks if you need a refresher) on the [osu!live Twitch channel](https://www.twitch.tv/osulive). If you can spare some time, you absolutely will not regret watching all the action as it went down!
+
+<iframe src="https://player.twitch.tv/?autoplay=false&video=v284639720" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="100%"></iframe>
+
+## Finals Week 2
+
+Having been defeated in last week's Winners' Bracket Final, Chile dropped to the Losers' Bracket at last, setting up a date with China for a spot in the Grand Final. China, who has themselves dropped to the Losers' Bracket after a Quarter Final tiebreaker loss to the United States, were in the ascendancy, having methodically crushed every other team in their path.
+
+Rampant 6-1 victories over the Netherlands and Hong Kong, a 7-1 victory over 2016 runners-up Indonesia, and even a 7-3 victory over three-time Champions South Korea headlined their intent at both revenge, and consequently, a third osu!catch World Cup win of their own.
+
+A comfortable opening victory on [Memme - Goat Fantasia](https://osu.ppy.sh/beatmapsets/634496#fruits/1346273) would not provide much solace for the Chileans, as the Chinese team strung together five victories on the bounce, including a humiliating triple fail capitulation for Chile early into the rarely-seen DoubleTime pick, [nanobii - HYPERDRIVE](https://osu.ppy.sh/beatmapsets/639991#fruits/1357221).
+
+Chile would secure a second point on [Morimori Atsushi - Paradigm Shift](https://osu.ppy.sh/beatmapsets/727816#fruits/1536733) courtesy of a typically gutsy one-miss from eldnl, but by this point the damage was done. China had secured victories in each of the four mod pools and left Chile nowhere to turn, eventually securing a 7-2 victory, as well as a rematch against the Americans for the ultimate prize.
```

Comment:
link to eldnl's new site profile for karma points

Reply:
Done this for all mentioned players